styles/variables-carbon.less

```diff
@@ -319,18 +319,29 @@
 // ---------------------------------------
 @agate-spinner-size: 52px;
 @agate-spinner-bg-padding: 9px;
+@agate-spinner-huge-size: (@agate-spinner-size * 2.5);
```

Comment:
based on the ratio with `IconButton`'s normal and huge sizes. Should we just set it as a value like `130px` instead?

Reply:
Yes. I believe that will be easier to understand. Imagine the scenario where we get new requirements and the normal spinner has to be 6px smaller. We didn't get any instruction to change the huge spinner, but now, because there's a relationship to the size of the normal spinner, now huge spinner gets smaller, if we don't notice the relationship.